CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Now the UI tests will hit API endpoints from a dockerised mock server.
 - Internal: Introduce TypeScript on non-critical components & deprecate FlowType.
 - Internal: Introduce integration tests for API endpoint integrations.
+- Internal: Added ScreenLayout component. This is currently used in the Welcome and Complete componenets.
```

Comment:
There's a typo here. But IMO Welcome and Complete should be described as "screens" or "screen components" rather than "components" for clarity since ScreenLayout is a component within them. 
```suggestion
- Internal: Added ScreenLayout component. This is currently used in the Welcome and Complete screens.
```